src/parser/element.js

```diff
@@ -8,7 +8,6 @@
 var g = require('../globalize');
 var assert = require('assert');
 var QName = require('./qname');
-var typeRegistry = require('./typeRegistry');
```

Comment:
any reason we choose to pass type registry through wsdl.js and not load here ?

Reply:
This does not work, because it introduces a circular dependency: `element.js` → `typeRegistry.js` → `element.js`